### DISCUSSION TITLE
DataHub not visible after installing

### DISCUSSION THREAD

**Discussion by cutec-chris** [pimcore/pimcore#10997] [Q&A / Support]
Hi,

we try to use DataHub and ImporterBundle on an plain skeleton installation of pimcore 10.2.5.
when we do this on an demo installation, the datahub symbol appears in the bar on the left.

```
/var/www/html# bin/console pimcore:bundle:list
+---------------------------------+---------+-----------+-------------+---------------+----------+
| Bundle                          | Enabled | Installed | Installable | Uninstallable | Priority |
+---------------------------------+---------+-----------+-------------+---------------+----------+
| AlepLdapBundle                  | ✔       | ✔         | ❌           | ❌             | ✔        |
| PimcoreDataHubBundle            | ✔       | ✔         | ❌           | ✔             | ✔        |
| PimcoreDataImporterBundle       | ✔       | ✔         | ❌           | ❌             | ✔        |
| PimcoreEcommerceFrameworkBundle | ✔       | ✔         | ❌           | ✔             | ✔        |
+---------------------------------+---------+-----------+-------------+---------------+----------+

Legend: I?: Can be installed? UI?: Can be uninstalled? UP?: Can be updated?
```

Do we need to configure something additionally ?

**Comment by jdreesen** [comment]
Did you try to clear the cache (`bin/console cache:clear`)?

**Reply by cutec-chris** [reply]
Ive done that from admin intrerface, but now again with the console command:
```
:/var/www/html# bin/console cache:clear

 // Clearing the cache for the dev environment with debug true                                                          

                                                                                                                        
 [OK] Cache for the "dev" environment (debug=true) was successfully cleared.                                            
                                                                                                                        
```

but nothing changes

**Reply by cutec-chris** [reply]
![image](https://user-images.githubusercontent.com/5136391/145546491-562d1a2b-111e-4621-975c-b46b31f3fcf3.png)

this appears in the console, when i make an conditianl breakpoint on those row i come out during loading the graphql classes

**Reply by cutec-chris** [reply]
Its the Data Importer Bundle, when i disable this it works.

**Reply by cutec-chris** [reply]
I have reinstalled everythiong clean now and it works.
The only difference is that i have installed the ldap bundle at last.
Before it was the first in the extension list.

**Reply by johanvinga** [reply]
Having the same issu... kinda dont want to reinstalll all of it tbh

**Reply by jdreesen** [reply]
We're having many issued with bundles not appearing too at the moment :/

**Reply by fashxp** [reply]
any root cause detectable?

**Reply by jdreesen** [reply]
I have no idea yet what the problem could be.

**Reply by langert** [reply]
I had to active the bundle in the config/bundles.php to be shown up. Activating via the backend was not enough...

Pimcore\Bundle\DataHubBundle\PimcoreDataHubBundle::class => ['all' => true],

**Comment by Mordef** [comment]
same here
`Pimcore\Bundle\DataHubBundle\PimcoreDataHubBundle::class => ['all' => true],`
not working

**Comment by SuryaWebfox** [comment]
Any news on this? I have installed the demo setup of Pimcore 11. It has the DataHub installed and configured, yet it is nowhere to be found in the menu.
